### PR TITLE
Added simple detection mechanism for synchronous return values

### PR DESCRIPTION
**Pull request by 0xricksanchez**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,8 @@
 		"fix": "npm run format:fix && npm run lint:fix"
 	},
 	"devDependencies": {
+		"@jazzer.js/core": "^1.2.0",
```
[the remark this answers is not in the document]
Comment (0xricksanchez) [reply package.json]: Actually no

File: packages/fuzzer/utils.cpp [hunk 2]
```diff
@@ -40,3 +47,61 @@ std::vector<std::string> LibFuzzerArgs(Napi::Env env,
...
 }
+
+// The following two small functions serve as a simple mechanism for keeping
+// track of encountered return values in the fuzzed target function. IFF both
+// `exclAsyncReturns` and `exclSyncReturns` are toggled the `MixedReturns` is
+// enabled. These toggles are used to inform the user about a potential
+// performance benefit when fuzzing asynchronously but only synchronous return
+// values are observed during a campaign. In such cases a user will be informed
+// about this once libfuzzer exits, e.g. due to a crash, or timeout.
+bool exclSyncReturns = false, exclAsyncReturns = false, MixedReturns = false;
```
[the remark this answers is not in the document]
Comment (0xricksanchez) [reply packages/fuzzer/utils.cpp]: Fixed naming

File: package.json [hunk 3]
```diff
@@ -35,7 +35,6 @@
 		"fix": "npm run format:fix && npm run lint:fix"
 	},
 	"devDependencies": {
-		"@jazzer.js/core": "^1.2.0",
 		"@jazzer.js/fuzzer": "^1.1.0",
```
Comment (bertschneider) [inline package.json:38]: There is still one `@jazzer` dependency left.

**Review by bertschneider — APPROVED** [review]
Great! Thx for taking care of this.

**Pull request merged**